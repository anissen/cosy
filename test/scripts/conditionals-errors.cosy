if 4 {
    print 'this does not have a boolean condition'
}