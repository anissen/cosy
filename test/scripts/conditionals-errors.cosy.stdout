â–  test/scripts/conditionals-errors.cosy, line 1:
1 | if 4 {
    ^^ The condition must evaluate to a be boolean value (instead of Num).
2 |     print 'this does not have a boolean condition'
3 | }

